### PR TITLE
Docs for zlayer

### PR DESCRIPTION
**Pull request by tampler**

### CODE REVIEW

File: docs/datatypes/has.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+id: datatypes_has
+title:  "Has"
+---
+The trait `Has[A]` is used with ZIO environment to express an effect'sdependency on a service of type `A`. <br>
```
Comment (holinov) [inline docs/datatypes/has.md]: `effect'sdependency` =>`effect's dependency`

File: docs/datatypes/zlayer.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  object ModuleA {
+    trait Service {
+      def letsGoA(v: Int): UIO[String]
+    }
+
+    val any: ZLayer[ModuleA, Nothing, ModuleA] =
+      ZLayer.requires[ModuleA]
+
+     val live: ZLayer.NoDeps[Nothing, ModuleA] = ZLayer.succeed {
```
Comment (holinov) [inline docs/datatypes/zlayer.md]: `ZLayer.NoDeps` is deprecated. use Layer

File: docs/datatypes/has.md (hunk 1 above)
Comment (tampler) [reply docs/datatypes/has.md]: Yes, thank you! Will update now

**Comment by tampler** [comment]
@adamgfraser  Pls merge this

**Comment by tampler** [comment]
@adamgfraser Updated

**Comment by adamgfraser** [comment]
Thanks! Can you merge upstream changes? There was an issue on `master` that is causing the build failures. Also looks like the CLA is still pending.

**Comment by tampler** [comment]
@adamgfraser Done

**Comment by tampler** [comment]
Adam, I granted it few times, but it won't pass any further. I also checked its page. Seems like it wants me to grant the CLA permission to all my repos, including private gists.

Why would I do that? Can you check CLA permissions ?

**Comment by adamgfraser** [comment]
@tampler When I go to the commit history it looks like there are two accounts that made commits, the `tampler` account (which has signed the CLA) and another one with just your name but apparently a different e-mail address associated with it. Not sure exactly what your setup is that led to that. If you just add that second e-mail to your Github account that should resolve the issue. Alternatively if you want to close and resubmit it should just work.

image

**Pull request closed without merging**